test/documentation/seqan3.css

```diff
@@ -131,3 +151,54 @@ table.doxtable th {
     color: black;
     font-style: bold;
 }
+
+/* automatic TOC */
+div.toc {
+    max-width: 20em;
+    width: auto;
+    align-content: center;
+}
+div.toc li {
+    font-size: 10pt;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
```

Comment:
this is only on macs?

Reply:
probably.. this is the default font in github so I just copy and pasted this.

Reply:
ok

Reply:
but this won't induce issues on non apple systems?

Reply:
it is already in the css for the normal page content, so does it? How does the documentation look like on your mac ? :wink:

Reply:
We'll see and fix it later if it is broken somewhere